test/browser_tests/conf.js

```diff
@@ -1,32 +1,118 @@
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    // Large Screen only tests
-    {
-      browserName:   'chrome',
-      specs:         [ 'spec_suites/large_screen/*.js' ],
-      chromeOptions: {
-        args: [ '--lang=en',
-                '--window-size=1200,900' ]
+var settings = require( './settings.js' );
+var defaultSuites = require( './default-suites.js' );
+
+function _paramIsSet( param ) {
+  return typeof param !== 'undefined';
+}
+
+/*
+* Params that need to be passed to protractor config.
+*/
+function _retrieveProtractorParams( params ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var parsedParams = {};
+
+  var checkParam = params.specs;
+  if ( _paramIsSet( checkParam ) ) {
+    parsedParams.specs = settings.specsBasePath + checkParam;
+  }
+
+  checkParam = params.browserName;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.browserName = checkParam; }
+
+  checkParam = params.version;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.version = checkParam; }
+
+  checkParam = params.platform;
+  if ( _paramIsSet( checkParam ) ) { parsedParams.platform = checkParam; }
+
+  return parsedParams;
+}
+
+function _copyParameters( params, capabilities ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var newCapabilities = [];
+  var injectParams = _retrieveProtractorParams( params );
+  var capability;
+  for ( var i = 0, len = capabilities.length; i < len; i++ ) {
+    capability = capabilities[i];
+    for ( var p in injectParams ) {
+      if ( injectParams.hasOwnProperty( p ) ) {
+        capability[p] = injectParams[p];
       }
     }
-  ],
+    capability.name = settings.testName +
+                      ' ' + capability.specs +
+                      ', running ' +
+                      capability.browserName +
+                      ' ' + capability.version +
+                      ' on ' + capability.platform +
+                      ' at ' +
+                      ( params.windowSize ?
+                        params.windowSize :
+                        settings.windowWidthPx + ',' +
+                        settings.windowHeightPx ) + 'px';
+    newCapabilities.push( capability );
+  }
+
+  return newCapabilities;
+}
+
+var config = {
+  baseUrl: settings.baseUrl,
 
-  baseUrl: env.baseUrl,
+  getMultiCapabilities: function() {
+    var params = this.params;
+    var capabilities = defaultSuites.essential;
+
+    // If --sauce=true flag was passed, configure Sauce Labs credentials.
+    if ( params.sauce === 'true' &&
+         process.env.SAUCE_USERNAME !== undefined &&
+         process.env.SAUCE_ACCESS_KEY !== undefined ) {
+      config.sauceSeleniumAddress = 'localhost:4445/wd/hub';
+      config.sauceUser = process.env.SAUCE_USERNAME;
+      config.sauceKey = process.env.SAUCE_ACCESS_KEY;
+    }
+
+    var newCapabilities = _copyParameters( params, capabilities );
+    if ( newCapabilities.length !== 0 ) {
+      capabilities = newCapabilities;
```

Comment:
yeah, that's how it works. I have a simplification refactor, but I'll push that in another PR after this one to avoid having to re-test the entire thing in the PR.